operations/network/management_network/hardware/ex2500.md

```diff
@@ -0,0 +1,51 @@
+# EX2500 installation and cabling
+
+This documentation helps network administrators and support personnel install and manage EX2500 cabinet in shasta management network.
+
+### HPE Cray EX2500 in quick summary:
+
+A single rack can accommodate up to 24 compute blade slots within 3 compute chassis.
+
+One rack supports the following:
+* Up to 3 compute chassis
+* Up to 3 power shelves with 4 rectifiers per shelf - 12 total 15kw rectifiers per rack. (Note: 1 rectifier per shelf is used for redundancy)
+* 3 PDUs (1 per power shelf)
+* 3 power input whips (3-phase)
+* Maximum of 24 compute blades
+* Maximum of 24 HPE Slingshot switch blades
+* Up to 2 in-rack Coolant distribution units for a max of 141KVA cooling capacity
+
+#### How does EX2500 connect to Shasta management network:
+
+Typically EX2500 cabinet connects in Shasta management network to the CDU switches (sw-cdu-00x).
+
+However in some smaller setups where CDU switches are not present such as "test and development", it is possible to connect the EX2500 cabinet to sw-leaf-bmc switch.
+
+#### How to connect EX2500 if CDU switches are not present:
+
+To connect EX2500 cabinet to a smaller system with out CDU switches will require a custom switch configuration from the network administrator.
+
+In a system without CDU switches, you can alternatively connect the EX2500 to sw-leaf-bmc switch.
+
+#### How to add EX2500 to Shasta management network:
+
+To add a EX2500 cabinet in your system, you will first need too create a "Mountain-Management" tab in SHCD and add it there. This will later on be used as your source to run the system seed files through CSI.
+
+The below is an example of how the entry in SHCD would look like.
+
```

Comment:
I think this needs a header that describes it as an example of the EX2500 when it includes a CDU switch.